cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -51,5 +51,7 @@
 
   public static final String ADAPTER_NAME = "adapterName";
```

Comment:
I keep it for now because the `ProgramLifecycleService.start` method is using it for updating `RunRecord` in store. Don't want to introduce too big of a refactoring around the lifecycle service in this PR.